src/main/java/frc/robot/Robot.java

```diff
@@ -73,7 +73,7 @@ public static enum Target {
     SPEAKER
   }
 
-  public static final RobotMode mode = Robot.isReal() ? RobotMode.REAL : RobotMode.SIM;
+  public static final RobotMode mode = Robot.isReal() ? RobotMode.REAL : RobotMode.REPLAY;
```

Comment:
Is this part of the gyro fix? What's the expectation here?

Reply:
no this just gets changed back and forth depending on if we want to run it in sim or replay it in the log

Reply:
Seems like a weird thing to commit then?